symforce/pybind/cc_values.cc

```diff
@@ -61,6 +61,20 @@ py::object PyAt(const sym::Valuesd& v, const sym::index_entry_t& index_entry) {
   return py::cast(v.At<T>(index_entry));
 }
 
+template <typename Scalar>
+py::object PyAtMatrix(const sym::Valuesd& v, const sym::index_entry_t& index_entry) {
+  const auto shape = EigenTypeShape(index_entry.type);
+  if (shape.first == 1 or shape.second == 1) {
```

Comment:
We need this branch to preserve what pybind was doing before, i.e. converting compile-time vectors to 1d numpy arrays.  I'm not thinking about whether I like that, just preserving behavior